### PR TITLE
Upgrading rds-broker and ginkgo

### PR DESCRIPTION
**Pull request by fearoffish**
What
----
The PR reflects the Update paas-rds-metric-collector story and an amount of replacing deprecated libraries.

How to review
----
1 - review code
2 - run local test as described in the README

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -27,20 +27,20 @@ var _ = Describe("collector", func() {
...
 
-		Eventually(rdsMetricsCollectorSession).Should(gexec.Exit(1))
+		Eventually(rdsMetricsCollectorSession, "5s").Should(gexec.Exit(1))
 		Expect(rdsMetricsCollectorSession.Err).To(gbytes.Say("Error loading config file"))
 		Expect(rdsMetricsCollectorSession.Err).To(gbytes.Say("no such file or directory"))
 	})
 
 	It("fails to start if the config is invalid", func() {
-		var err error
+		//var err error
```
Comment (corlettb) [inline main_test.go:36]: Remove comment

File: suite_test.go [hunk 2]
```diff
@@ -1,10 +1,11 @@
 package main_test
 
 import (
+	// "fmt"
```
Comment (corlettb) [inline suite_test.go:4]: remove comment

**Review by corlettb — CHANGES_REQUESTED** [review]
remove a couple of comments and I think we are good to go.

File: main_test.go (hunk 1 above)
Comment (malcgds) [reply main_test.go]: Done

File: suite_test.go (hunk 2 above)
Comment (malcgds) [reply suite_test.go]: Done

**Review by corlettb — APPROVED** [review]

**Pull request merged**